### ISSUE TITLE
ETA? Obfuscation options when downloading or updating?

### ISSUE THREAD

**Issue by baimafeima** [snapcrafters/tor-browser#5]
What is the ETA of this snap? Do you have any mechanisms at place when downloading or updating the snap? I am asking because my current attempts with https://github.com/micahflee/torbrowser-launcher have failed. Access to the Tor network is blocked in China. Is there any chance you could implement an obfuscation option like obfs4, ECC/XOR, and/or stunnel in the actual snap updating and downloading mechanism?

**Comment by popey** [comment]
We should probably remove this snap as it's not working and we'd rather have working snaps in snapcrafters. Unless someone else has some time to work on it.

**Comment by chpio** [comment]
> Is there any chance you could implement an obfuscation option like obfs4, ECC/XOR, and/or stunnel in the actual snap updating and downloading mechanism?

No, but you could post a feature request to [flatpak](https://github.com/flatpak/flatpak)

**Comment by Ads20000** [comment]
Isn't @oSoMoN or someone working on a Firefox snap? Presumably the Tor Browser snap would work with minimal changes to that `snapcraft.yaml`, when there is one that works for Firefox...
